protocol/triple/triple-tool/gen/generator/tripleTpl_test.go

```diff
@@ -319,24 +332,24 @@ type GreetService_GreetClient interface {
 	Close() error
 }
 
-type greetServiceGreetClient struct {
+type GreetServiceGreetClient struct {
```

Comment:
Is **GreetService_GreetClient** redundant? There is no need to use this interface.